tests/unit/AboutTheBusiness.spec.ts

```diff
@@ -0,0 +1,57 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import { shallowMount } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import { useBusinessStore } from '@/stores'
+import AboutTheBusiness from '@/components/AgmExtension/AboutTheBusiness.vue'
+import mockRouter from './mockRouter'
+import { AgmExtEvalIF } from '@/interfaces'
+import { DateUtilities } from '@/services'
+
+Vue.use(Vuetify)
+Vue.use(VueRouter)
+
+const vuetify = new Vuetify({})
+setActivePinia(createPinia())
+const businessStore = useBusinessStore()
+
+describe('About The Business component', () => {
+  const router = mockRouter.mock()
+
+  it('handles empty data', async () => {
+    const wrapper = shallowMount(AboutTheBusiness, {
+      vuetify,
+      router,
+      propsData: { data: null }
+    })
+    await Vue.nextTick()
+
+    expect(wrapper.find('#entity-in-good-standing').exists()).toBe(false)
+    expect(wrapper.find('#entity-date-of-incorporation').exists()).toBe(false)
```

Comment:
Why would the elements not exist?

Reply:
I think what happens is that since the `data` prop is required, passing `null` to it does not create the component. [EntityHeader.spec.ts](https://github.com/bcgov/business-filings-ui/blob/ba9afd9d7ce0d4a1339c4f15f099da2be8baf41d/tests/unit/EntityHeader.spec.ts#L24) has a similar test in it, but perhaps a better way of doing this is to check that the component elements do not exist (like `v-card`).

Reply:
Ah, I see.

You probably don't need to test that the prop is not provided, as that will result in a compile error. I suggest testing with an empty data object (either `EmptyAgmExtEval` or `{}`).

Reply:
Fixed and looks good. Thanks!